docs/rationale.md

```diff
@@ -22,7 +22,7 @@ Prettier maintains the way your string is escaped. For example, `"🙂"` won’t
 
 ### Empty lines
 
-It turns out that empty lines are very hard to automatically generate. The approach that Prettier takes is to preserve empty lines the way they were in the original source code. There are two additional rules:
+It turns out that empty lines are very hard to automatically generate. The approach that Prettier takes is to preserve empty lines the way they were in the original source code, in a similar fashion to preserving [comments](#comments). There are two additional rules:
```

Comment:
```suggestion
It turns out that empty lines are very hard to automatically generate. The approach that Prettier takes is to preserve the existence of empty lines the way they were in the original source code, in a similar fashion to preserving [comments](#comments). There are two additional rules:
```

adding "the existence of" to *hopefully* add clarification

Reply:
Has this issue been resolved?